roles/pulibrary.nginxplus/files/conf/http/plantain-staging.conf

```diff
@@ -0,0 +1,68 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
Should `proxy_pass http://pulfa3-staging;` be to `proxy_pass http://plantain-staging;` (given that `upstream` is `plantain-staging`)?

Reply:
nice catch.